bin/flume-ng

```diff
@@ -211,6 +211,10 @@ run_flume() {
   if [ ${CLEAN_FLAG} -ne 0 ]; then
     set -x
   fi
+  
+  if [[ $FLUME_JAVA_OPTS =~ "Xmx" ]]; then
```

Comment:
I think it might be good to rename the `$JAVA_OPTS` to `$JAVA_HEAP_MAX` or add this as a new variable (as in Hadoop: https://github.com/apache/hadoop-common/blob/trunk/hadoop-common-project/hadoop-common/src/main/bin/hadoop-functions.sh#L544).
The reason is that if in the future new default parameters will be added to the `$JAVA_OPTS` passing in `-Xmx` will clear them too, resulting in unexpected behaviour (unless we do this change when we add the new parameters).